api/src/main/java/graphql/nadel/Nadel.java

```diff
@@ -69,7 +70,8 @@
     final Map<String, StitchingDsl> stitchingDsls;
     final ServiceExecutionFactory serviceExecutionFactory;
     final List<Service> services;
-    final GraphQLSchema overallSchema;
+    final GraphQLSchema privateOverallSchema;
```

Comment:
naming matters - its not private - LTE NOT NOT use this

Its meant to be the schema for "querying" and "introspection"